contracts/view.cairo

```diff
@@ -93,7 +94,7 @@ func save_all_non_expired_options_with_premia_to_array{syscall_ptr: felt*, peder
 
         return save_all_non_expired_options_with_premia_to_array(
             lptoken_address,
-            array_len_so_far + OptionWithPremia.SIZE,
+            array_len_so_far + 1,
```

Comment:
now the array length is not x*7, but x because of element in the array being OptionWithPremia